src/main-menu.ts

```diff
@@ -1,24 +1,98 @@
-import { LitElement, PropertyValueMap, css, html } from "lit";
-import { customElement, state } from "lit/decorators.js";
+import { LitElement, css, html } from "lit";
+
+import { customElement, property, state } from "lit/decorators.js";
 import { game } from "./game";
-import { AudioController } from "./audio/audio_controller";
 
 @customElement("main-menu")
 export class MainMenu extends LitElement {
   static styles = css`
     :host {
-      width: 100vw;
-      height: 100vh;
+      width: 100%;
+      aspect-ratio: 16 / 9;
+      position: relative;
       display: flex;
       align-items: center;
       justify-content: center;
       flex-direction: column;
+      background: rgb(240, 200, 98);
+    }
+    h1 {
+      font-family: "Poppins", sans-serif;
+      font-size: 12vw;
+      position: absolute;
+      margin: 0;
+      top: 5%;
+      color: #d69b04;
+    }
+    #play-button,
+    #settings-button,
+    #start-button,
+    #back-button {
+      position: absolute;
+      background: none;
+      border: none;
+      font-size: 3vw;
+      font-weight: bold;
+      color: #644903;
+      cursor: pointer;
+      opacity: 60%;
+      transition: all 0.25s ease-in-out;
+    }
+    #start-button:hover,
+    #back-button:hover {
+      opacity: 100%;
+    }
+    #play-button {
+      left: 50%;
+      top: 50%;
+      padding: 0 15vw 15vw 15vw;
+      transform: translate(-50%);
+    }
+    #settings-button {
+      right: 8%;
+      top: 35%;
+      padding-bottom: 25vw;
+    }
+    :host([play-hovered]) #play-button,
+    #play-button:hover {
```

Comment:
nit: we could merge all the hover rules into one group e.g.
```
    #start-button:hover,
    #back-button:hover,
    :host([play-hovered]) #play-button,
    #play-button:hover,
    :host([settings-hovered]) #settings-button,
    #settings-button:hover {
      opacity: 100%;
    }
```

Reply:
At this stage, I almost wonder if we should be using a class lol

Reply:
Yeah look in retrospect it's so wild i didn't just use a class lol. updated.